content/blog/keda-2.0-release.md

```diff
@@ -0,0 +1,147 @@
++++
+title = "Announcing KEDA 2.0 - Taking app autoscaling to the next level"
+date = 2020-11-14
+author = "KEDA Maintainers"
++++
+
+A year ago, we were excited to **announce our 1.0 release with a core set of scalers**, allowing the community to start autoscaling Kubernetes deployments.  We were thrilled with the response and encouraged to see many users leveraging KEDA for event driven and serverless scale within any Kubernetes cluster.
+
+With KEDA, any container can scale to zero and burst scale based directly on event source metrics.
+
+![Logo](./../../img/logos/keda-horizontal-color.png)
+
+While KEDA was initially started by Microsoft & Red Hat **we have always strived to be an open & vendor-neutral project** in order to support everybody who wants to scale applications.
+
+That's why earlier this year we donated and were accepted as a CNCF Sandbox project. We feel this is a strong signal to the community to fully align with the CNCF open mindset and vendor neutrality.
+
+**Since KEDA 1.0 was released we’ve been growing** with new scalers for many different sources including IBM MQ, Postgres, and Huawei Cloudeye, supporting new authentication providers such as HashiCorp Vault, and constantly improving the user experience to make application auto scaling dead-simple.
+
+Today, we are happy to announce another milestone - **KEDA 2.0 is now generally available and ready to scale all your workloads!** 🎊
+
+## Getting started
+
+There are many ways to get started with KEDA:
+
+- Install with [OperatorHub.io](https://operatorhub.io/operator/keda)
+
+- Install with Helm:
+
+>$ helm repo add kedacore https://kedacore.github.io/charts
```

Comment:
nit: might look better to squash those 3 commands into on block

Reply:
Done

Reply:
Done